packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -137,6 +140,14 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
         commands.registerCommand({ id: 'workbench.action.files.openFolder' }, {
             execute: () => commands.executeCommand(WorkspaceCommands.OPEN_FOLDER.id)
         });
+        commands.registerCommand({ id: 'default:type' }, {
+            execute: args => {
+                const editor = this.monacoEditors.current;
```

Comment:
that's an internal field for inline editors, you should not use it

instead use `MonacoEditor.getCurrent(EditorManager)`

Reply:
done